kafka-client/src/main/java/dev/responsive/kafka/internal/db/mongo/KVDoc.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2023 Responsive Computing, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dev.responsive.kafka.internal.db.mongo;
+
+import org.bson.types.ObjectId;
+
+public class KVDoc {
```

Comment:
I'm not super familiar with mongo, but I take it they refer to a key-value entry as a "doc" (document?)

Reply:
MongoDB only has "documents" -- they don't even directly expose the "key", which is an `ObjectId`. I still need to figure out exactly how Mongo works with this regard, but essentially yes everything that's inserted into Mongo is considered a "document"